docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -110,7 +110,7 @@ Valid options for `[Container]` are listed below:
 | PodmanArgs=--add-host foobar     | --add-host foobar                      |
 | PublishPort=true                 | --publish                              |
 | ReadOnly=true                    | --read-only                            |
-| RemapGid=keep-id                 | --group-add keep-id                    |
+| RemapGid=0:20000:400             | --group-add 0:20000:400                |
```

Comment:
What I meant in [my comment](https://github.com/containers/podman/issues/17908#issuecomment-1489145830): Shouldn't the `podman run equivalent` be `--gidmap` instead of `--group-add`? At least this is what quadlet uses in the `RemapUsers=manual` case:

https://github.com/containers/podman/blob/60d16d9f8e292a54b87d16eb0d782512b0cfd41a/pkg/systemd/quadlet/quadlet.go#L907-L914